src/components/analytics/analytics.service.ts

```diff
@@ -0,0 +1,11 @@
+import * as analyticsModel from './analytics.model'
```

Comment:
We're just exporting the `interface` and the `model`, we can import them separately, I just thought that it would be better to use analyticsModel as a prefix `analyticsModel.model` or `analyticsModel.IAnalytics`